### PR TITLE
v4.7 changelog

### PR DESCRIPTION
**Pull request by jasonblais**
We should include a note that `Zoom plugin now supports an on-premise Zoom server.`.

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v4.7
+
+Release date: 2017-02-16
```
Comment (sudheerDev) [inline source/administration/changelog.md]: Should this be 2018 :) ?

Comment (jasonblais) [reply source/administration/changelog.md]: Yes! Thank you :)

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,9 +4,154 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Added a new endpoint called `/users/tokens/search`. This endpoint gets all tokens for all users if one has the `manage_system` permission.
+
+#### Plugin API Changes (Beta)
+
+#### Plugin Hook Changes (Beta)
+
+### Database Changes
+
+### WebSocket Event Changes
+
```
Comment (saturninoabril) [inline source/administration/changelog.md]: - Added `delete_team` web socket event to notify client whenever a team is deleted (e.g. via API call)

Reference:
- https://github.com/mattermost/mattermost-api-reference/pull/328/files
- https://github.com/mattermost/mattermost-server/pull/8167

Comment (amyblais) [reply source/administration/changelog.md]: Thank you, added.

File: source/administration/changelog.md [hunk 3]
```diff
@@ -4,9 +4,161 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - Fixed timestamp not clickable in desktop mobile view.
+ - Fixed deleting a Team via the API breaks the Web UI.
+ 
+### Compatibility
+
+#### Removed and Deprecated Features
+
+- All API v3 endpoints have been deprecated, and scheduled for removal in Mattermost v5.0.
+- The permanent query parameter of the DELETE `/teams/{team_id}` APIv4 endpoint for permanently deleting a team is scheduled for removal in Mattermost v4.7. // XXX Jason needs to update
+- The `mentionKeys` prop in post type plugins is now removed to fix case sensitive mention highlighting. Plugins can retrieve the `mentionKeys` prop from the store as needed.
```
Comment (jasonblais) [inline source/administration/changelog.md]: Added a minor breaking change for plugins

File: source/administration/changelog.md [hunk 4]
```diff
@@ -4,9 +4,166 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v4.7
+
+Release date: 2018-02-16
+
+### Security Update
+
+- Mattermost v4.7.0 contains multiple security fixes ranging from low to high severity. [Upgrading](http://docs.mattermost.com/administration/upgrade.html) is highly recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 14 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/).
```
Comment (jasonblais) [inline source/administration/changelog.md]: Is it multiple?

File: source/administration/changelog.md [hunk 5]
```diff
@@ -4,9 +4,166 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+#### config.json
+
+Multiple setting options were added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+
+ - Under `ServiceSettings` in `config.json`:
+    - Added `"ImageProxyType": ""`, `"ImageProxyOptions": ""`, and `"ImageProxyURL": ""` to ensure posts served to the client will have their markdown modified such that all images are loaded through a proxy when these keys are configured.
+    - Added `"ExperimentalGroupUnreadChannels": true` as `default_on` or `"ExperimentalGroupUnreadChannels": false` as `disabled` settings to show an unread channel section in the webapp sidebar.
+    - Added `"VaryByUser": false`, a user-based rate limiting, to rate limit on token and on userID. // XXXX This is under `RateLimitingSettings`
```
Comment (jasonblais) [inline source/administration/changelog.md]: Should be under `RateLimitingSettings`, not `ServiceSettings`

File: source/administration/changelog.md [hunk 6]
```diff
@@ -4,9 +4,166 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+    - Added `"ExperimentalEnableDefaultChannelLeaveJoinMessages": true` that allows disabling of leave/join messages in the default channel, usually Town Square.
+
+### API Changes
+
+ - It is required that any new integrations use API v4 endpoints. For more details, and for a complete list of available endpoints, see [https://api.mattermost.com/](https://api.mattermost.com/).
+ - All API v3 endpoints have been deprecated, and scheduled for removal in Mattermost v5.0.
+
+#### RESTful API v4 Changes
```
Comment (jasonblais) [inline source/administration/changelog.md]: Propose writing this section consistently with previous versions. E.g. https://docs.mattermost.com/administration/changelog.html#restful-api-v4-changes

File: source/administration/changelog.md [hunk 7]
```diff
@@ -4,9 +4,166 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+ - Added auto lowercase team and channel names in API requests. This ensures that the channel name is automatically lowercased for endpoints taking team or channel names as URL parameters.
+ - Added `POST /emoji/search`, `GET /emojis/name/{emoji_name}`, and `GET /emoji/autocomplete` API endpoints. These ensure that the benefits of `GET` for important performance related actions such as autocompleting are included.
+ - Added a new endpoint called `/users/tokens/search`. This endpoint gets all tokens for all users if one has the `manage_system` permission.
+
+#### Plugin API Changes (Beta)
```
Comment (jasonblais) [inline source/administration/changelog.md]: Were these reviewed by a dev?

File: source/administration/changelog.md [hunk 8]
```diff
@@ -4,9 +4,166 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+#### Plugin Hook Changes (Beta)
```
Comment (jasonblais) [inline source/administration/changelog.md]: Were these reviewed by a dev?

File: source/administration/changelog.md [hunk 9]
```diff
@@ -4,9 +4,166 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+### Database Changes
```
Comment (jasonblais) [inline source/administration/changelog.md]: Were these reviewed by a dev?

**Comment by jasonblais** [comment]
Otherwise looks good, removing assignment,

File: source/administration/changelog.md (hunk 4 above)
Comment (amyblais) [reply source/administration/changelog.md]: There are 2 in the spreadsheet.

File: source/administration/changelog.md (hunk 6 above)
Comment (amyblais) [reply source/administration/changelog.md]: Fixed

File: source/administration/changelog.md (hunk 5 above)
Comment (amyblais) [reply source/administration/changelog.md]: Fixed

File: source/administration/changelog.md [hunk 10]
```diff
@@ -4,9 +4,171 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+ - It is required that any new integrations use API v4 endpoints. For more details, and for a complete list of available endpoints, see [https://api.mattermost.com/](https://api.mattermost.com/).
+ - All API v3 endpoints have been deprecated, and scheduled for removal in Mattermost v5.0.
+
+#### RESTful API v4 Changes
+
+ - Added `GetChannelByName` and `GetTeamByName` to auto lowercase team and channel names in API requests. This ensures that the channel name is automatically lowercased for endpoints taking team or channel names as URL parameters.
+ - Added `POST /emoji/search`, `GET /emojis/name/{emoji_name}`, and `GET /emoji/autocomplete` to add consistency with user search/autocomplete endpoints. These API endpoints ensure that the benefits of `GET` for important performance related actions such as autocompleting are included.
+ - Added `/users/tokens/search` to allow System Admin to be able to find, manage and revoke personal access tokens as needed. This endpoint gets all tokens for all users if one has the `manage_system` permission.
+
+#### Plugin API Changes (Beta)
```
Comment (esethna) [inline source/administration/changelog.md]: @AndersonWebStudio needs to populate these?

**Comment by jasonblais** [comment]
@AndersonWebStudio I'm going to merge this. If there are any API or Webhook changes that aren't included in the changelog, you can submit a separate PR for them.

**Pull request merged**